frontend-react/src/pages/About.jsx

```diff
@@ -1,7 +1,76 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { motion } from 'framer-motion';
 
-const About = () => (
-  <div>About</div>
-);
+import image from '../assets/index';
+import { urlFor, client } from '../client';
+
+const About = () => {
+  const aboutContainer = [
+    {
+      title: 'Web Designer ',
+      description: 'I am a web designer with a passion for creating beautiful and functional web applications.',
+      imgURL: image.about01,
+    },
+    {
+      title: 'Frontend Developer ',
+      description: 'I am a frontend developer with a passion for creating beautiful and functional web applications.',
+      imgURL: image.about02,
+    },
+    {
+      title: 'Backend Developer ',
+      description: 'I am a backend developer with a passion for creating beautiful and functional web applications.',
+      imgURL: image.about03,
+    },
+  ];

```

Comment:
Create a constants folder and move this data to it. You can import it here from the constants folder. 
**It's a good practice to separate the dummy or any constant data from the component logic.**